### ISSUE TITLE
Add ansible-later

### ISSUE THREAD

**Issue by xoxys** [awesome-devops/awesome-ansible#15]
Would you be interested in adding [ansible-later](https://github.com/xoxys/ansible-later) to your list? It's a fork of [ansible-review](https://github.com/willthames/ansible-review) with some benefits and an alternative best practice checker to ansible-lint.

Regards
Robert

**Comment by KeyboardInterrupt** [comment]
Yes, totally!, Feel free to open up an PR, or anyone for that matter :+1:
